extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -31611,100 +31774,99 @@
+	'tooltip-pt-mycontris' => 'Listagem de suas contribuições',
+	'tooltip-pt-login' => 'Você é encorajado a autenticar-se, apesar disso não ser obrigatório.',
+	'tooltip-pt-anonlogin' => 'Você é encorajado a autenticar-se, apesar disso não ser obrigatório.',
+	'tooltip-pt-logout' => 'Sair',
 	'tooltip-ca-talk' => 'Discussão sobre o conteúdo da página',
-	'tooltip-ca-edit' => 'Pode editar esta página.
-Utilize o botão "Antever resultado" antes de gravar, por favor.',
-	'tooltip-ca-addsection' => 'Iniciar uma nova secção',
-	'tooltip-ca-viewsource' => 'Esta página está protegida; só pode ver o conteúdo.',
+	'tooltip-ca-edit' => 'Você pode editar esta página. Use o botão "Mostrar previsão" antes de salvar.',
+	'tooltip-ca-addsection' => 'Iniciar uma nova seção',
+	'tooltip-ca-viewsource' => 'Esta página está protegida.
+Você pode, no entanto, visualiar seu código-fonte.',
 	'tooltip-ca-history' => 'Edições anteriores desta página.',
 	'tooltip-ca-protect' => 'Proteger esta página',
-	'tooltip-ca-unprotect' => 'Alterar a protecção desta página',
+	'tooltip-ca-unprotect' => 'Alterou a proteção desta página',
 	'tooltip-ca-delete' => 'Apagar esta página',
 	'tooltip-ca-undelete' => 'Restaurar edições feitas a esta página antes da eliminação',
 	'tooltip-ca-move' => 'Mover esta página',
-	'tooltip-ca-watch' => 'Adicionar esta página à lista de páginas vigiadas',
+	'tooltip-ca-watch' => 'Adicionar esta página à sua lista de páginas vigiadas',
 	'tooltip-ca-unwatch' => 'Remover esta página da lista de páginas vigiadas',
-	'tooltip-search' => 'Pesquisar nesta wiki',
-	'tooltip-search-go' => 'Ir para uma página com este nome exacto, caso exista',
-	'tooltip-search-fulltext' => 'Procurar páginas que contêm este texto',
-	'tooltip-p-logo' => 'Visite a página principal',
-	'tooltip-n-mainpage' => 'Visitar a página principal',
-	'tooltip-n-mainpage-description' => 'Visitar a página principal',
-	'tooltip-n-portal' => 'Sobre o projecto',
-	'tooltip-n-currentevents' => 'Informação temática sobre acontecimentos actuais',
-	'tooltip-n-recentchanges' => 'A lista de mudanças recentes nesta wiki.',
-	'tooltip-n-randompage' => 'Carregar página aleatória',
-	'tooltip-n-help' => 'Um local reservado para auxílio.',
-	'tooltip-t-whatlinkshere' => 'Lista de todas as páginas que contêm links para esta',
-	'tooltip-t-recentchangeslinked' => 'Mudanças recentes nas páginas para as quais esta contém links',
-	'tooltip-feed-rss' => "''Feed'' RSS desta página",
-	'tooltip-feed-atom' => "''Feed'' Atom desta página",
-	'tooltip-t-contributions' => 'Ver as contribuições deste utilizador',
-	'tooltip-t-emailuser' => 'Enviar uma mensagem de correio a este utilizador',
-	'tooltip-t-upload' => 'Upload de ficheiros',
+	'tooltip-search' => 'Pesquisar em {{SITENAME}}',
+	'tooltip-search-go' => 'Ir a uma página com este exato nome, caso exista',
+	'tooltip-search-fulltext' => 'Procurar páginas que contenham este texto',
+	'tooltip-p-logo' => 'Acessar a página principal',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Acessar a página principal" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)